scripts/version updates/update_meilisearch_version.sh

```diff
@@ -0,0 +1,295 @@
+#!/bin/bash
+
+## Constants
+
+RED="\033[0;31m"
+BRED="\033[1;31m"
+GREEN="\033[0;32m"
+BGREEN="\033[1;32m"
+YELLOW="\033[0;33m"
+BYELLOW="\033[1;33m"
+BLUE="\033[0;34m"
+BBLUE="\033[1;34m"
+PINK="\033[0;35m"
+BPINK="\033[1;35m"
+CYAN="\033[0;36m"
+BCYAN="\033[1;36m"
+GREY="\033[0;37m"
+BGREY="\033[1;37m"
+WHITE="\033[0m"
+NC="\033[0m"
```

Comment:
```suggestion
BRED="\033[1;31m"
BGREEN="\033[1;32m"
BYELLOW="\033[1;33m"
BBLUE="\033[1;34m"
BPINK="\033[1;35m"
WHITE="\033[0m"
NC="\033[0m"
```
Wouldn't we remove the ones we don't need? No specific advice, it's up to you

Reply:
I thought about it, i felt like we could keep them in case we wanted to add some label. But they can always be added afterwards